contracts/truffle.js

```diff
@@ -1,7 +1,10 @@
 const HDWalletProvider = require("truffle-hdwallet-provider")
+const PrivateKeyProvider = require('truffle-privatekey-provider')
```

Comment:
Not a huge deal, but it looks like we may not need this package. This note appears on a similar functioning package [truffle-hdwallet-provider-privkey](https://www.npmjs.com/package/truffle-hdwallet-provider-privkey)
> If you are using Web3 1.0.0, you do not need this package! There is a function to get an account object from a private key. The reason I made this package was because this function doesn't exist in the legacy Web3 library.